test_regress/t/t_clk_concat.pl

```diff
@@ -10,16 +10,14 @@
 
 scenarios(simulator => 1);
 
-my $out_filename = "$Self->{obj_dir}/V$Self->{name}.xml";
+my $out_filename = "$Self->{obj_dir}/V$Self->{name}.tree.json";
 
 compile(
-    verilator_flags2 => ["+define+ATTRIBUTES"],
+    verilator_flags2 => ["+define+ATTRIBUTES --no-json-edit-nums"],
     );
 
 if ($Self->{vlt_all}) {
-    file_grep("$out_filename", qr/\<var loc="e,74,.*?" name="clk0" .*dir="input" .*vartype="logic" origName="clk0" clocker="true" public="true"\/\>/i);
-    file_grep("$out_filename", qr/\<var loc="e,75,.*?" name="clk1" .*dir="input" .*vartype="logic" origName="clk1" clocker="true" public="true"\/\>/i);
-    file_grep("$out_filename", qr/\<var loc="e,76,.*?" name="clk2" .*dir="input" .*vartype="logic" origName="clk2" clocker="true" public="true"\/\>/i);
+    files_identical($out_filename, $Self->{golden_filename});
```

Comment:
Most of the tests that had file_greps were to look for very specific items in the dump, so I'd prefer to generally preserve these.  My concern with just looking for identical dumps is that it looses what's important that the test is looking for, making it more likely that someone making a change will just "accept" the new golden file, and not realize the test is no longer effective.